### PR TITLE
Extract Styleguide

### PR DESCRIPTION
**Pull request by appinteractive**
I have extracted the Styleguide so it will not be part of that repo anymore which leads to faster build times and smaller docker builds.

## Todo`s

- [x] Fix docker build

### CODE REVIEW

**Comment by Lulalaby** [comment]
Still WIP?

**Comment by appinteractive** [comment]
Not really but would wait a bit as there are some changes what need to be merged into the new repo from some prs

File: package.json [hunk 1]
```diff
@@ -10,8 +10,8 @@
     "start": "cross-env node server/index.js",
     "generate": "nuxt generate",
     "lint": "eslint --ext .js,.vue .",
-    "styleguide": "cd ./styleguide && yarn dev",
-    "styleguide:build": "cd ./styleguide && yarn build:lib && cd ../",
+    "styleguide": "echo 'Command styleguide is Deprecated!'",
```
Comment (roschaefer) [inline package.json]: Why not simply remove? Do we have dependencies in our CI/CD pipeline that would break?

Comment (appinteractive) [reply package.json]: I was thinking about user that still typing that commands ^^

**Pull request merged**